apps/examples/testcase/le_tc/kernel/tc_mqueue.c

```diff
@@ -118,6 +118,10 @@ static mqd_t g_timedrecv_mqfd;
 static int enter_notify_handler = 0;
 static int timedsend_check = 0;
 static int timedreceive_check = 0;
+#ifdef CONFIG_DISABLE_PTHREAD
```

Comment:
Do we need to keep pthread on mqueue tc? It looks not clear and not simple due to ```#ifdef```. Because this is a mqueue tc, we can remove pthread usages.